.github/workflows/stage_release.yaml

```diff
@@ -132,3 +137,26 @@ jobs:
           SONATYPE_TOKEN_USERNAME: ${{ vars.KROXYLICIOUS_SONATYPE_TOKEN_USERNAME }}
           SONATYPE_TOKEN_PASSWORD: ${{ secrets.KROXYLICIOUS_SONATYPE_TOKEN_PASSWORD }}
         run: ./scripts/transition-staging-repository-state.sh -s close
+
+      - name: 'Comment on PR to prompt next phase of the release'
+        if: ${{ github.event.inputs.dry-run == 'false' }}
+        env:
+          GH_TOKEN: ${{ secrets.KROXYLICIOUS_RELEASE_TOKEN }} # For the gh cmd line tool used by stage-release.sh
+        run: |
+          gh pr comment ${{ env.WORK_BRANCH }} --body-file - << 'EOF'
+          Hey, that's the ${{ github.event.inputs.release-version }} release prepared and its artefacts successfully staged to Sonatype. This PR contains a commit for ${{ github.event.inputs.release-version }} and a commit for ${{ github.event.inputs.development-version }}
+          which will reopen main for new work.
+          
+          There are a couple of steps to do before the release is completed.
+          
+          * [ ] Let the CI workflow on this PR complete.
+          * [ ] [Kick the tyres](https://github.com/kroxylicious/kroxylicious-junit5-extension/blob/main/RELEASING.md#making-the-release-public)
```

Comment:
We said we want to automate this part, but let's leave that out of scope from this PR.